config/config.defaults.yml

```diff
@@ -285,6 +285,7 @@ default: &default_settings
   external_search_offset_min: 0
 
   # Harmony service API connection
+  harmony_api_in_use: false
```

Comment:
Why did you feel we need this boolean separately from simply checking if the api_url is set?